app/utils/asyncInjectors.js

```diff
@@ -1,10 +1,35 @@
+import invariant from 'invariant';
 import createReducer from '../reducers';
 
+export const isType = type => target => typeof target === type;
+export const isFunction = isType('function');
```

Comment:
i actually checked last night—lodash isn't explicitly included in our dependencies, but it is in our devDeps... I can move into dependencies and use destructured imports (for tree shaking) if you are ok with it?